apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js

```diff
@@ -124,17 +124,22 @@ const RequestAssignment = ({ issue }) => {
 
 RequestAssignment.propTypes = issueShape
 
-const HoursInput = styled(TextInput.Number)`
-  height: 32px;
+const HoursInput = styled(TextInput.Number).attrs({
+  mode: 'strong',
+  step: '1',
+  min: '0',
+  max: '1000',
```

Comment:
The HoursInput component was written by someone else in the FundIssue component. I simply copied it over to the RequestAssignment to maintain a consistent appearance and behaviour. I'd be happy to make any changes you want across the app but I'm not in a position to say why any decisions were made for this component.

Reply:
Yeah, I am neither on that position to take on these decision, is just 1000 seems also a lot for me, maybe 500 could be equally valid? 300? I dunno...

I know @stellarmagnet recommended to avoid limiting for now, but I think a limit makes sense, just a more appropriate one, or even 1000 if we consider that is the appropriate.
Removing the limit does not seem a good idea for me, at least in this case I think it does not represent a block or issue in the code, since this field is just an estimation.